net/http/http_network_session.h

```diff
@@ -155,6 +155,12 @@ struct NET_EXPORT HttpNetworkSessionParams {
   // If true, HTTPS URLs can be sent to QUIC proxies.
   bool enable_quic_proxies_for_https_urls = false;
 
+#if defined(STARBOARD)
+    // If true, request to an origin without recorded alt-svc info will
+    // try to establish both QUIC and TCP connections and use the faster one.
+    bool use_quic_for_unknown_origins;
```

Comment:
This doesn't seem to be used anywhere, only set. Is this scaffolding for a followup that makes it work?